### ISSUE TITLE
No sound in lock screen

### ISSUE THREAD

**Issue by rubenxi** [halverneus/org.kde.video#15]
Is it supposed to play the audio on the lock screen or is it a failure? I mean, I can get the video, but not the audio in lock screen, in desktop mode it works all perfectly.

**Comment by halverneus** [comment]
Out of curiosity, if you have music playing and then lock your screen, does the music continue playing? Admittedly, I've never tried playing audio in either case, but I see nothing in the code that would treat the lock screen any different than a wallpaper. When I get some free time I'll dig out some headphones and try it out on my machine and see if I get the same behavior.

**Comment by rubenxi** [comment]
> Out of curiosity, if you have music playing and then lock your screen, does the music continue playing? Admittedly, I've never tried playing audio in either case, but I see nothing in the code that would treat the lock screen any different than a wallpaper. When I get some free time I'll dig out some headphones and try it out on my machine and see if I get the same behavior.

Yes, if I have smplayer, vlc or any program playing music, it still plays it even if I lock the screen.
